backbone/src/main/java/org/researchstack/backbone/step/CompletionStep.java

```diff
@@ -10,7 +10,7 @@
 public class CompletionStep extends InstructionStep {
 
     /* Default constructor needed for serilization/deserialization of object */
-    CompletionStep() {
+    public CompletionStep() {
```

Comment:
Need to create a custom step in the MPower project that extended InstructionStep.